erizo/src/erizo/rtp/BandwidthEstimationHandler.cpp

```diff
@@ -229,19 +223,39 @@ void BandwidthEstimationHandler::pickEstimator() {
 }
 
 void BandwidthEstimationHandler::sendREMBPacket() {
+  sink_ssrc_ = 0;
+  source_ssrcs_.clear();
+  ELOG_DEBUG("Update MediaStream SSRCs");
+  connection_->forEachMediaStream([this] (const std::shared_ptr<MediaStream> &media_stream) {
+    ELOG_DEBUG("MediaStream %s, publisher %u, sink %u, source %u", media_stream->getId().c_str(),
+    media_stream->isPublisher(), media_stream->getVideoSinkSSRC(), media_stream->getVideoSourceSSRC());
+    if (media_stream->isReady() && media_stream->isPublisher()) {
+      sink_ssrc_ = media_stream->getVideoSinkSSRC();
+    }
+    source_ssrcs_.push_back(media_stream->getVideoSourceSSRC());
+  });
+
+  if (sink_ssrc_ == 0) {
+    ELOG_WARN("No SSRC available to send REMB");
```

Comment:
I wonder if there could be cases where we should not notify with a warning here.